server/src/lib/data.ts

```diff
@@ -0,0 +1,95 @@
+import {db} from "./db"
+import BlogSchema from "../Schema/Blog"
+import {z} from "zod"
+import { PrismaClient, Category } from '@prisma/client';
+
+
+// getting all the available blogs in the databse 
+export async function getBlog(){
```

Comment:
this function returns multiple blog, the name should be `getBlogs()` to prevent misunderstanding.